electron/src/index.ts

```diff
@@ -1,25 +1,79 @@
-import { app, BrowserWindow, ipcMain, shell, autoUpdater } from 'electron'
+import {
+  app,
+  BrowserWindow,
+  ipcMain,
+  shell,
+  autoUpdater,
+  Menu,
+  dialog,
+} from 'electron'
 import * as contextMenu from 'electron-context-menu'
 import * as path from 'path'
 import * as fs from 'fs'
-// import log from 'electron-log'
 import initAgent, {
   StateSignal,
   STATUS_EVENT,
 } from '@lightningrodlabs/electron-holochain'
 
+import { devOptions, prodOptions, stateSignalToText } from './holochain'
 import {
-  devOptions,
-  projectsHappPath,
-  prodOptions,
-  stateSignalToText,
   BINARY_PATHS,
-} from './holochain'
-import {
+  DATASTORE_PATH,
   INTEGRITY_VERSION_NUMBER,
+  KEYSTORE_PATH,
   PREV_VER_USER_DATA_MIGRATION_FILE_PATHS,
+  PROJECTS_HAPP_PATH,
   USER_DATA_MIGRATION_FILE_PATH,
 } from './paths'
+import defaultMenu = require('electron-default-menu')
```

Comment:
this line gives mixed signals. I think it should stick with import

Reply:
i had to enable a flag in tsconfig to allow importing it normally, since it uses the old `exports =` syntax. let me know if this is undesirable. I also had the change the call signature of `contextMenu()` after adding this flag.